### PR TITLE
a403464238804805 empty asset list

### PR DESCRIPTION
**Pull request by Tuoris**

### CODE REVIEW

File: openregistry/lots/basic/tests/lot_blanks.py [hunk 1]
```diff
@@ -314,21 +314,55 @@ def create_single_lot():
...
+    empty_asset_lot = deepcopy(self.initial_data)
+    empty_asset_lot.update({
+        'status': 'draft',
+        'assets': [],
+    })
+    response = self.app.post_json(
+        '/lots',
+        {"data": empty_asset_lot},
+    )
+    self.assertEqual(response.status, '201 Created')
```
Comment (yarsanich) [inline openregistry/lots/basic/tests/lot_blanks.py]: check `content_type`.

Comment (Tuoris) [inline openregistry/lots/basic/tests/lot_blanks.py]: Fixed

**Comment by Tuoris** [comment]
Closed due to irrelevance.

**Comment by Tuoris** [comment]
Closed due to irrelevance.

**Pull request closed without merging**